packages/ove-app-maps/README.md

```diff
@@ -101,4 +101,4 @@ All considerations when using this app are directly related to its reliability a
 
 1. The maps app tends to load many tiles on screens with higher resolutions and tile servers that are slow and remote tend to perform very poorly compared to servers that are much faster and locally hosted.
 2. JavaScript executed to load custom overlays must not introduce any performance bottlenecks or execute code that has a negative impact on the reliability of OVE.
-3. The Map Layers configuration must be available (unless it is embedded in `config.json`) before the server-side of this app is launched using PM2 or Docker.
+3. The [Map Layers configuration](../ove-app-maps/docs/MAP_LAYERS_JSON.md) must be available (unless it is embedded in `config.json`) before the server-side of this app is launched using PM2 or Docker.
```

Comment:
"If the [Map Layers configuration](../ove-app-maps/docs/MAP_LAYERS_JSON.md) is specified in the `config.json` file as a URL (rather than being directly embedded in the file), then this must be available before the server-side of this app is launched using PM2 or Docker."

Reply:
Good catch.

Reply:
Reworded to "If the [Map Layers configuration](../ove-app-maps/docs/MAP_LAYERS_JSON.md) is specified as a URL (rather than being directly embedded in the `config.json` file), then this must be available before the server-side of this app is launched using PM2 or Docker."

Note that a URL can be provided within this file or via a separate environment variable.